doc/api/async_hooks.md

```diff
@@ -4,22 +4,12 @@
 
 > Stability: 1 - Experimental
 
-The `async_hooks` module provides an API to register callbacks tracking the
-lifetime of asynchronous resources created inside a Node.js application.
-It can be accessed using:
-
-```js
-const async_hooks = require('async_hooks');
-```
```

Comment:
For consistency with other docs, the `It can be accessed using...` and subsequent code block should not be removed.